estate/models/estate_property.py

```diff
@@ -0,0 +1,40 @@
+from datetime import timedelta
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'Estate testing model'
+
+
+    name = fields.Char('Title', required=True)
+    description = fields.Text('Description')
+    postcode = fields.Char('Postcode')
+    date_availability = fields.Date('Available From', copy=False, default=fields.Date.today() + timedelta(days=90))
+    expected_price = fields.Float('Expected Price', default=2, required=True)
+    selling_price = fields.Float('Selling Price', readonly=True)
+    bedrooms = fields.Integer('Bedrooms')
+    living_area = fields.Integer('Living Area (sqm)')
+    facades = fields.Integer('Facades')
+    garage = fields.Boolean('Is Garage')
+    garden = fields.Boolean('Is garden')
+    garden_area = fields.Integer('Garden Area')
+    garden_orientation = fields.Selection([
+        ('north', 'North'),
+        ('south', 'South'),
+        ('east', 'East'),
+        ('west', 'West')
+    ], string='Garden Orientation')
+    active = fields.Boolean(default=True)
+    state = fields.Selection([
+        ('new', 'New'),
+        ('offer_received', 'Offer Received'),
+        ('offer_accepted', 'Offer Accepted'),
+        ('sold', 'Sold'),
+        ('cancelled', 'Cancelled')
+    ], string='Select State', required=True, copy=False, default='new')
+    property_type_id = fields.Many2one(comodel_name='estate.property.type', string='Property Type')
```

Comment:
```suggestion
    property_type_id = fields.Many2one('estate.property.type', string='Property Type')
```
No need of adding comodel_name